Week3/vehicleList.js

```diff
@@ -0,0 +1,28 @@
+// Make a list of vehicles, you can add "motorbike", "caravan", "bike", or more.
+
+// How do you get the third element from that list?
+
+// Change the function vehicle to use the list of question 7. So that vehicle("green", 3, 1) prints "a green new bike".
+
+ let vehicles = [' car', ' motorbike', ' bike', ' caraven' ];
+
+function vehicleType(color,code, age){
+    if (age < 1) {
```

Comment:
here the same remark as above (about the bad practice to modify an argument that is passed in the function)